.github/workflows/_reusable.build-flutter-example.yml

```diff
@@ -13,15 +13,19 @@ permissions:
 
 jobs:
   flutter-build-example-ios:
-    runs-on: macos-11
+    runs-on: hetzner-mac-pm
     timeout-minutes: 30
     steps:
+      - name: Clean up workspace
+        shell: bash
+        working-directory: ${{ github.workspace }}
+        run: rm -rf *
```

Comment:
🤔  Similar to what I wrote above. I don't know if "working directory" accepts an empty value or what it does with it, but I suppose it's worth making sure we're not deleting things we don't want to delete, like the home dir.

Reply:
We can actually remove `working-directory` because the default value is the path of the workspace.  😅